packages/lambda-powertools-datadog-metrics/README.md

```diff
@@ -0,0 +1,77 @@
+# dazn-datadog-metrics
```

Comment:
I think this should be `lambda-powertools-datadog-metrics` - there are a few instances of this in the PR.

Reply:
thanks, will get these changed